scribbl/app/assets/stylesheets/posts.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the posts controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
avoid using generators when possible - this helps me avoid that "coloring within the lines" feeling